cmd/deploy/deploy.go

```diff
@@ -659,6 +693,28 @@ func (dc *DeployCommand) deployDivert(ctx context.Context, opts *Options) error
 	return nil
 }
 
+func (dc *DeployCommand) deployIngress(ctx context.Context, ingress *ingresses.Ingress, c *ingresses.Client) error {
```

Comment:
Can we move this into a new file and test it?

Reply:
This file has 1k lines, it's hard to look for code inside it, but I'm ok to have this function on this file

Reply:
I know, but there are many other functions within this file that makes more sense to move than this one. And if we move this, I would move all the functions of the `DeployCommand` struct.